homework-submission/week1/src/components/Todos.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import List from "./List";
+import Checkbox from "./Checkbox";
+import RemoveTodo from "./RemoveTodo";
+
+class Todos extends Component {
+  render() {
+    return (
+      <div>
+        <ul>
+          {this.props.items.map((item, id) => (
+            <div key={id} className={item.done? 'checked' : ''}>
+            <Checkbox 
```

Comment:
Here too, you have a `div` inside of a `ul` which will make the following tree with what is inside the List Component:
```HTML
<ul>
  <div>
    <div>
      <li>
      </li>
    </div>
  </div>
</ul>
```